test/core/vitest.config.ts

```diff
@@ -46,6 +46,7 @@ export default defineConfig({
     exclude: ['**/fixtures/**', ...defaultExclude],
     slowTestThreshold: 1000,
     testTimeout: 2000,
+    retry: 3,
```

Comment:
It looks like this configuration will cause all core tests to be retried three times.